parser.go

```diff
@@ -381,6 +381,19 @@ func (r *Rule) option(key item, l *lexer) error {
 		r.Vars[name] = v
 		lastContent := r.Contents[len(r.Contents)-1]
 		lastContent.Options = append(lastContent.Options, &ContentOption{Name: key.value, Value: strings.Join(parts, ",")})
+	case key.value == "flowbits":
+		nextItem := l.nextItem()
+		parts := strings.Split(nextItem.value, ",")
+		fb := &Flowbit{}
+		if len(parts) == 2{
```

Comment:
This code should be refactored to use a guard clauses and return any errors in the flowbits definition, something like this:

https://play.golang.org/p/cSj5NFGLUj4